configure.ac

```diff
@@ -44,7 +44,6 @@ AC_CHECK_HEADERS([fcntl.h glob.h stdlib.h sys/time.h errno.h limits.h])
 # Checks for dependencies needed by ctags
 AC_CHECK_HEADERS([fnmatch.h direct.h io.h sys/dir.h])
 AC_DEFINE([HAVE_STDBOOL_H], [1], [whether or not to use <stdbool.h>.])
-AC_CHECK_HEADERS(dirent.h,have_dirent_h=yes)
```

Comment:
I only looked at this one teeny part, but `dirent.h` is still included in `main.c` in Geany and upstream. Is the intention to just permanently guard out it's inclusion via `HAVE_DIRENT_H` never being defined? Feel free to ignore me, it just looked funny.

Reply:
It wasn't there before - I added it in https://github.com/geany/geany/pull/2132/commits/361c058b3f727a3d4cda9e4908b588f98f151f83 and then because of this https://github.com/universal-ctags/ctags/pull/2100 it could be dropped again.